StPetersburg/RNBefns7.xml

```diff
@@ -24,136 +24,139 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+
+                  <msItem xml:id="ms_i1.4">
+                     <locus from="81r" to="81r"/>
                      <title type="complete" ref="LIT4663Eqabanni"/>
-                     <note> the prayer was received from the sky by a nobleman in the city of Franks, <foreign xml:lang="gez"> ʾafrǝngi</foreign>, 
-                the indication in <bibl> 
-                           <ptr target="bm:Turaev1906rukopisi"/>
-                           <citedRange unit="page">23</citedRange> 
-                        </bibl> 
+                     <note>The prayer that was received from the sky by
+                       a nobleman "in the city of Franks" (<foreign xml:lang="gez">ʾafrǝngi</foreign>).
                      </note>
                   </msItem>
-            
                </msContents>
-          
+
                <physDesc>
                   <objectDesc form="Codex">
                      <supportDesc>
                         <support>
                            <material key="parchment"/>
                            <material key="paper"/>
                         </support>
-                
+
                         <extent>
                            <measure unit="leaf">88</measure>
                            <dimensions type="outer" unit="mm">
                               <height>18</height>
                               <width>16</width>
                               <depth/>
                            </dimensions>
-                           <note>the indication of size in <bibl> 
-                                 <ptr target="bm:Turaev1906rukopisi"/>
-                                 <citedRange unit="page">22</citedRange> 0,18x0,16 cm must be a mistake </bibl>
+                           <note>The indication of size in <bibl>
+                                 <ptr target="bm:Turaev1906rukopisi"/><citedRange unit="page">22</citedRange> as
+                                 0,18x0,16 cm must be a mistake.</bibl>
                            </note>
-                           <note> Four initial leaves (to be numbered i-iv) are made of paper, left blank, have no numbers </note>
+                           <note>Four initial leaves are unnumbered blank paper leaves (here numbered i-iv).</note>
                         </extent>
-                
+
                         <foliation>
-                           The leaves of the first paper quire (i-iv) have no numbers. Pagination on verso-sides of the folios in <foreign xml:lang="Coptic"/>, from  112 to 195 </foliation>
-                        <collation> The first four unnombered leaves (to be counted as i-iv) are made of paper </collation>
-                
+                           The leaves of the first paper quire (i-iv) have no numbers. Pagination on verso-sides of the folios
+                            in <foreign xml:lang="Coptic"/>, from  112 to 195.</foliation>
```

Comment:
I think the element foreign should contain the coptic text, not be used as empty element before that text, if the squares here are the coptic

Reply:
ok. actually i do not see Coptic numbers any longer.